packages/base/spec.gts

```diff
@@ -40,13 +40,704 @@ import StackIcon from '@cardstack/boxel-icons/stack';
 import AppsIcon from '@cardstack/boxel-icons/apps';
 import LayoutList from '@cardstack/boxel-icons/layout-list';
 import Brain from '@cardstack/boxel-icons/brain';
+import { tracked } from '@glimmer/tracking';
 
 export type SpecType = 'card' | 'field' | 'app' | 'skill';
 
 class SpecTypeField extends StringField {
   static displayName = 'Spec Type';
 }
 
+class Isolated extends Component<typeof Spec> {
+  @tracked icon: CardOrFieldTypeIcon | undefined;
```

Comment:
I had to move out the `Isolated`, `Fitted`, and `Edit` views to satisfy the type check since I use `@tracked`.